src/routes/session/session-get.ts

```diff
@@ -2,13 +2,39 @@ import { NextFunction, Request, Response } from "express";
 
 export const SessionGet = (req: Request, res: Response, next: NextFunction) => {
 	if (!req.params[0]) {
-		return next(new Error("Missing redirect url for session.  Needs to be in format `/session/:redirectUrl`"));
+		return next(new Error("Missing redirect url for session. Needs to be in format `/session/:redirectUrl`"));
 	}
 
-	return res.render("session.hbs", {
+	res.render("session.hbs", {
 		title: "Logging you into GitHub",
 		APP_URL: process.env.APP_URL,
 		redirectUrl: new URL(req.params[0], process.env.APP_URL).href,
 		nonce: res.locals.nonce
 	});
 };
+
+export const SessionEnterpriseGet = (req: Request, res: Response, next: NextFunction) => {
+	if (!req.params[0]) {
+		return next(new Error("Missing redirect url for session enterprise. Needs to be in format `/session/enterprise/:redirectUrl`"));
+	}
+
+	res.render("gitHub-loading.hbs", {
+		APP_URL: process.env.APP_URL,
+		redirectUrl: new URL(req.params[0], process.env.APP_URL).href,
+		nonce: res.locals.nonce,
+		isGitHubCloud: false
+	});
+};
```

Comment:
Not sure I understand the need to have a different one between cloud and enterprise.  Aren't they doing the same thing?

Reply:
Both are loading pages, but I guess they will have different text, so separated them

Reply:
I guess since we have `isGitHubCloud: false` here, we can do the different text in hbs via this flag?

Reply:
- `isGitHubCloud` removed.
- Replaced by the content `text` instead.